runtime/criusupport/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ target_link_libraries(j9criu
 		dl
 		j9utilcore
 		j9pool
+		omrutil
```

Comment:
Is this change required?

Reply:
It's needed for `getOMRVMThreadName` and `releaseOMRVMThreadName`

Reply:
Looks like it's not needed anymore now that j9utilcore is added as a target.